src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -137,6 +149,19 @@ protected JestElasticsearchClient(Map<String, String> props, JestClientFactory f
             .preemptiveAuthTargetHosts(address.stream()
                 .map(addr -> HttpHost.create(addr)).collect(Collectors.toSet()));
       }
+
+      if (secured) {
+        log.info("Using secured connection");
+        SslFactory kafkaSslFactory = new SslFactory(Mode.CLIENT, "nonev", false);
+        kafkaSslFactory.configure(config.sslConfigs());
+        SSLContext sslContext = kafkaSslFactory.sslContext();
+        SSLConnectionSocketFactory sslSocketFactory = new SSLConnectionSocketFactory(
+            sslContext, SSLConnectionSocketFactory.getDefaultHostnameVerifier());
+
+        builder.sslSocketFactory(sslSocketFactory); // this only affects sync calls
```

Comment:
nit: comments on separate line are better to distinguish between code changes and comment changes on git history